geomstats/learning/kmeans.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, riemannian_metric, n_clusters=8, init='random',
         self.riemannian_metric = riemannian_metric
         self.tol = tol
         self.verbose = verbose
+        self.point_type = point_type
```

Comment:
Adding point_type as an attribute may not be necessary once you have created your mean method